pkg/util/shell/shell_test.go

```diff
@@ -0,0 +1,48 @@
+package shell
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+type testCase struct {
+	command        string
+	expectedOutput string
+	stdout         *bytes.Buffer
+}
+
+func TestShellCat(t *testing.T) {
+	testCases := []testCase{
+		{
+			command: "cat testdata/data.txt",
+			expectedOutput: func() string {
+				if os.Getenv("GOOS") == "windows" {
+					return "Hello DevSpace!\r\n"
+				}
+				return "Hello DevSpace!\n"
+			}(),
+			stdout: &bytes.Buffer{},
+		},
+		{
+			command: "echo 123 | cat",
+			expectedOutput: func() string {
+				if os.Getenv("GOOS") == "windows" {
+					return "123\r\n"
+				}
+				return "123\n"
+			}(),
+			stdout: &bytes.Buffer{},
+		},
```

Comment:
create a failing test case please for a not found file

Reply:
done